unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs

```diff
@@ -120,10 +121,11 @@ public void AddBookmark()
         }
     }
 
+    // TODO: FIX THIS
     public void ReportScene()
     {
         var coords = playerCoords.Get();
-        WebInterface.SendReportScene($"{coords.x},{coords.y}");
+        WebInterface.SendReportScene(DCL.Environment.i.world.state.GetSceneNumberByCoords(coords));
```

Comment:
would be better to inject `Environment.i` through controller's constructor

Reply:
done!